src/helpers/cosmos.ts

```diff
@@ -433,16 +457,16 @@ export class WalletWrapper {
       'code_id',
     ]);
 
-    return attributes[0].code_id;
+    return parseInt(attributes[0].code_id);
```

Comment:
```suggestion
    const codeId = parseInt(attributes[0].code_id, 10);
    if (isNaN(codeId)) {
       throw new Error('shit, codeid is fucked up');
    }
    return codeId;
```
but I'd prefer move changes like that into separate PR as out of scope.

Reply:
Yup 100% agree both on changes proposed and moving them as out of scope :)